pkg/kubectl/scale.go

```diff
@@ -41,7 +41,7 @@ type Scaler interface {
 	Scale(namespace, name string, newSize uint, preconditions *ScalePrecondition, retry, wait *RetryParams) error
 	// ScaleSimple does a simple one-shot attempt at scaling - not useful on its own, but
 	// a necessary building block for Scale
-	ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) error
+	ScaleSimple(namespace, name string, preconditions *ScalePrecondition, newSize uint) (string, error)
```

Comment:
At the very least you need to name the return parameters here in the interface, where people will be looking for documentation!